x-pack/plugins/security_solution/public/common/utils/route/helpers.ts

```diff
@@ -23,3 +25,11 @@ export const RouterSpyStateContext = createContext<[RouteSpyState, Dispatch<Rout
   initRouteSpy,
   () => noop,
 ]);
+
+const detectionsPaths = [ALERTS_PATH, `${RULES_PATH}/id/:id`, `${CASES_PATH}/:detailName`];
+
+export const isDetectionPage = (pathname: string) =>
```

Comment:
this function already exists [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/helpers.tsx#L158-L163). could you please narrow it down to one or the other?

Reply:
The implementations are different. I extracted this function to a shared place, but because I am not sure anymore if it is actually reusable I will roll back my changes.